components/feature/search/src/main/assets/search/list.json

```diff
@@ -477,7 +477,7 @@
     "lij": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "bing", "amazon-it", "ddg", "wikipedia-lij"
+          "google-b-m", "bing", "ddg", "wikipedia-lij"
```

Comment:
I'm not sure why `amazon-it` is being removed here. Was that asked for?

Reply:
No this was not requested.  However, I didn't find amazon-it in `it-IT` but in this section for Ligurian.  So I was assuming that it's a mistake.  So I removed it here and added to the `it-IT` section.

However, looking into this more just now I found that Ligurian is just north-western Italy.  This was a mistake to remove it.  I'll put this back in.  Thanks,